### PR TITLE
Create code-of-conduct.md

### PR DESCRIPTION
**Pull request by jthomasmock**
Add code of conduct

### CODE REVIEW

File: code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+**Community Guidelines:**
+
+* **Be friendly and patient**: Remember you might not be communicating in someone else's primary spoken or programming language, and others may not have your level of understanding.
+* **Be welcoming**: Our communities welcome and support people of all backgrounds and identities. This includes, but is not limited to members of any race, ethnicity, culture, national origin, color, immigration status, social and economic class, educational level, sex, sexual orientation, gender identity and expression, age, size, family status, political belief, religion, and mental and physical ability.
+Harassment includes:
+* **Be respectful:** We are a world-wide community of professionals, and we conduct ourselves professionally. Disagreement is no excuse for poor behavior and poor manners. 
+
+**Disrespectful and unacceptable behavior includes, but is not limited to:**
+* Violent threats or language.
```
Comment (MikeBadescu) [inline code-of-conduct.md:17]: duplicate, I would remove, listed below in more detail

File: code-of-conduct.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+* Discriminatory or derogatory jokes and language.
+* Posting sexually explicit or violent material.
+* Posting, or threatening to post, people's personally identifying information ("doxing").
+* Insults, especially those using discriminatory terms or slurs.
+* Behavior that could be perceived as sexual attention.
+* Advocating for or encouraging any of the above behaviors.
+* Offensive comments related to gender, gender identity and expression, sexual orientation, disability, mental illness, neuro(a)typicality, physical appearance, body size, age, race, or religion.
+* Unwelcome comments regarding a person's lifestyle choices and practices, including those related to food, health, parenting, drugs, and employment.
+* Deliberate misgendering or use of 'dead' or rejected names.
+* Gratuitous or off-topic sexual images or behaviour in spaces where they're not appropriate.
```
Comment (MikeBadescu) [inline code-of-conduct.md:27]: IMO, I would keep `Posting sexually explicit or violent material` above, it is more general

File: code-of-conduct.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+* Threats of violence.
+* Incitement of violence towards any individual, including encouraging a person to commit suicide or to engage in self-harm.
+* Deliberate intimidation.
+* Stalking or following.
+* Harassing photography or recording, including logging online activity for harassment purposes.
+* Sustained disruption of discussion.
+* Unwelcome sexual attention.
+* Pattern of inappropriate social contact, such as requesting/assuming inappropriate levels of intimacy with others.
+* Continued one-on-one communication after requests to cease.
+* Deliberate "outing" of any aspect of a person's identity without their consent except as necessary to protect vulnerable people from intentional abuse.
```
Comment (MikeBadescu) [inline code-of-conduct.md:38]: ~~I think this is covered under "doxing" above - or maybe we should combine them (?)~~

File: code-of-conduct.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+* Publication of non-harassing private communication.
+
+**On Slack:**
+
+* Avoid using overtly sexual or offensive usernames or profile photos which might detract from a friendly, safe and welcoming environment for all.
+* Do not publish text/screenshots of anything shared in Slack without explicit consent from the author. This includes screenshots of private messages to public channels, as well as conversations on public channels to anywhere outside of Slack.
```
Comment (MikeBadescu) [inline code-of-conduct.md:44]: I am not sure that I understand the last bullet point. Private messages are covered above ("publication of non-harassing private communication"). I perceive Slack to be the same as MeetUp website, Twitter, or Linked group, or GitHub - all the  publicly accessible info is public. Our Slack is not "invite only" (i.e. the invite link is public), anybody can join. Also, we should have a consistent policy across all online spaces, MeetUp, LinkedIn, etc. For this reason think we should remove the Slack section. Thoughts?

Comment (MikeBadescu) [reply code-of-conduct.md:44]: We also need to address reporting. Maybe having a link to the LT on MeetUp? (We need to update the LT page too)

Comment (dallasrug) [reply code-of-conduct.md:44]: Probably has to do with consent to publish material that is not their own

**Pull request merged**